hercules-ci.nix

```
@@ -17,5 +17,23 @@
     };
   };
 
-  herculesCI.ciSystems = [ "x86_64-linux" "x86_64-darwin" ];
+  herculesCI = herculesArgs: {
+    onPush.plutus-ledger-api-publish = {
```

```suggestion
    onPush.default = {
```

If you have other checks that validate the API, but aren't `packages.plutus-ledger-api-rust-src` or a dependency, it may be beneficial to extend the default job instead, so that you make use of the implicit dependency, that effects only run when all builds are ok. That way you don't push a broken release and if that's considered acceptable, you might even re-tag after fixing the problem.